tool/templates/cpp2/method_impl.h.jinja

```diff
@@ -0,0 +1,29 @@
+inline {##}
+{{- m.return_ty }} {##}
```

Comment:
Note: when the open or close tag has a `-` it means to strip whitespace before or after it. The `{##}` is a comment that protects whitespace.